js/components/operatorSignIn/operatorSignInModal.tsx

```diff
@@ -94,6 +96,7 @@ const OperatorSignInModalContent = ({
   close: () => void;
 }): ReactElement => {
   const [badge, setBadge] = useState<BadgeEntry | null>(null);
+  const [radio, setRadio] = useState<string>("");
```

Comment:
thought: I don't love how this component still requires a radio state just for tryAgain. But I don't see a different way to do that, so 👍

Reply:
yeah, I hear you. try again could probably be changed to happen at the network level, but that has other downsides and is a little tricky.